libraries/AP_Soaring/Variometer.cpp

```diff
@@ -9,9 +9,8 @@ Manages the estimation of aircraft total energy, drag and vertical air velocity.
 Variometer::Variometer(const AP_Vehicle::FixedWing &parms) :
     _aparm(parms)
 {
-    _climb_filter = LowPassFilter<float>(1.0/60.0);
-
-    _vdot_filter2 = LowPassFilter<float>(1.0f/60.0f);
+    _climb_filter.set_cutoff_frequency(1.0/60);
+    _vdot_filter2.set_cutoff_frequency(1.0/60);
```

Comment:
Aren't these redundant given the in-header initialisation?

(well, for `_climb_filter` - shouldn't the init of `_vdo_filter` be the same, 'though?

Reply:
fixed